sdk/web-pubsub/web-pubsub-client-protobuf/rollup.config.mjs

```diff
@@ -0,0 +1,5 @@
+import config from "@azure/dev-tool/shared-config/rollup.js";
+const { makeConfig } = config;
+import pkg from "./package.json" assert { type: "json" };
```

Comment:
Same comment as above that since we're on Node 16 for now, we'd have to use a flag to build this properly.

Reply:
Went with a simple `readFile` for now in https://github.com/Azure/azure-sdk-for-js/pull/27377/commits/a74fe46ad58ddea4173c3c7261bdc0652a7bfba3